deploy/handler/role.yaml

```diff
@@ -118,6 +128,18 @@ rules:
   - configmaps
   verbs:
   - "*"
+- apiGroups:
+  - authentication.k8s.io
+  resources:
+  - tokenreviews
+  verbs:
+  - create
+- apiGroups:
+  - authorization.k8s.io
+  resources:
+  - subjectaccessreviews
+  verbs:
+  - create
```

Comment:
why is this needed in this PR?

Reply:
Need by prometheus integration at nmstate-handler